read-fonts/src/collections.rs

```diff
@@ -2,3 +2,6 @@
 
 pub mod int_set;
 pub use int_set::IntSet;
+
+pub mod range_set;
```

Comment:
I'm not sure if we need to make the module public? If there are only a very small number of items we could export them all.